extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -723,7 +723,7 @@ export class AmpA4A extends AMP.BaseElement {
           // arrayBuffer is null, force collapse.
           if (!fetchResponse || !fetchResponse.arrayBuffer ||
               fetchResponse.headers.has('amp-ff-empty-creative')) {
-            this.forceCollapse();
+            this.attemptCollapse();
```

Comment:
So it turns out this is a misnomer as this actually calls local function named forceCollapse which in turn calls applyNoContentUI which calls attemptCollapse (see https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/0.1/amp-ad-ui.js#L54).